src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -264,6 +264,18 @@
                                 </table>
                             </f:repeatable>
                         </f:entry>
+                        <f:entry title="${%Abort new patch sets}"
```

Comment:
I tried to keep this change small but I agree, it is the better way to solve this. I'll change it.